cmdstanpy/__init__.py

```diff
@@ -1,10 +1,14 @@
 import atexit
+import os
 import os.path
 import shutil
 import tempfile
 
+if 'CMDSTAN' in os.environ:
+    CMDSTAN_PATH = os.environ['CMDSTAN']
+else:
+    CMDSTAN_PATH = os.path.abspath(os.path.join('.', 'releases', 'cmdstan'))
```

Comment:
> I'm not sure should we install anything manually under site-packages

Doh!!!!  please forgive my being new to Python.  I was only thinking about case for install via GitHub.

> have multiple cmdstan in "default" location

yes, the intent here is that whenever there's a new release of CmdStan, the user can rerun this script and it will get the new release, leaving previous releases in place.